src/components/HelpResourceCard.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Link,
+  CardGroup,
+} from "@trussworks/react-uswds";
+
+export const HelpResourceCard = () => {
+  return (
+    <CardGroup>
+      <Card>
+        <CardHeader>
+          <h2>Help & Resources</h2>
+        </CardHeader>
+
+        <CardBody>
+          <p>
+            Links to Overview, Related Resources, FAQs, Release Notes and
+            Contact Us
+          </p>
+        </CardBody>
+
+        <CardFooter>
+          <ul className="usa-list">
+            <li>
+              <Link
+                href={"https://www.epa.gov/airmarkets/overview"}
```

Comment:
I think this should be a react-router-dom <Link> and we should link to the pages that don't exist yet instead of the epa.gov sites. 

https://v5.reactrouter.com/web/api/Link
https://v5.reactrouter.com/web/guides/quick-start